Refresh "Apply a provisioning package" article

aisgbnok opened the pull request:
## Summary

As noted in #10462, some of the articles/pages relating to provisioning packages are out of date. One such article is "[Apply a provisioning package](https://docs.microsoft.com/windows/configuration/provisioning-packages/provisioning-apply-package)." I have spent a few hours going through the flow in a Windows 11 VM, taking screenshots, and then updating the page.

Closes #10462 

### Details

- Documentation updated to provide a Windows 11 first experience. (Breadcrumbs/paths reflect Windows 11 instead of 10.) However, the majority of the flow and documentation work for non EOL versions of Windows 10 as well.
- Consolidated redundant "Apply a provisioning package" sections in other articles.
   - "[Set up a multi-app kiosk on Windows 10 devices](https://docs.microsoft.com/windows/configuration/lock-down-windows-10-to-specific-apps#apply-ppkg)" and "[Set up a shared or guest PC with Windows 10/11](https://docs.microsoft.com/windows/configuration/set-up-shared-or-guest-pc#apply-the-provisioning-package)," has a section "Apply provisioning package to device" and it is largely the same as the main article. Attempted to remove the redundancy and link to the main article.
- Small grammatical and spelling improvements [such as replacing "comdlet" with "cmdlet" in Tip.](https://docs.microsoft.com/windows/configuration/lock-down-windows-10-to-specific-apps#apply-ppkg)

### Questions & Improvements

Here are a few questions and concerns I have.

- [ ] Used low-resolution screenshots, I have higher resolution should I provide them?
- [ ] Consolidate Notes from other articles
   - [ ] Note in "Set up a multi-app kiosk on Windows 10 devices" seems to be specific to that page, but I am not familiar with kiosk and am unsure. https://github.com/MicrosoftDocs/windows-itpro-docs/blob/8fd28257835108c53b64193dba40c7f09db50d59/windows/configuration/lock-down-windows-10-to-specific-apps.md?plain=1#L546-L547
   - [ ] Note in "Set up a shared or guest PC with Windows 10/11" seems specific to that page or at least a certain configuration/policy such as [CleanPC CSP](https://docs.microsoft.com/windows/client-management/mdm/cleanpc-csp)? It could possibly be moved to the main page, unsure? https://github.com/MicrosoftDocs/windows-itpro-docs/blob/8fd28257835108c53b64193dba40c7f09db50d59/windows/configuration/set-up-shared-or-guest-pc.md?plain=1#L190-L191
- [x] [Microsoft Settings URIs](https://docs.microsoft.com/windows/uwp/launch-resume/launch-settings-app) for some of the breadcrumbs. https://github.com/MicrosoftDocs/windows-itpro-docs/blob/8fd28257835108c53b64193dba40c7f09db50d59/windows/configuration/provisioning-packages/provisioning-apply-package.md?plain=1#L37

Please let me know if there is anything else. Thanks!

JohanFreelancer9 commented on windows/configuration/lock-down-windows-10-to-specific-apps.md:
```diff
@@ -541,42 +541,10 @@ Use the Windows Configuration Designer tool to create a provisioning package. [L
 ### Apply provisioning package to device
 
-Provisioning packages can be applied to a device during the first-run experience (out-of-box experience or "OOBE") and after ("runtime").
+Provisioning packages can be applied to a device during initial setup (out-of-box experience or "OOBE") and after ("runtime"). For more information, see "[Apply a provisioning package](./provisioning-packages/provisioning-apply-package.md)."
```
```suggestion
Provisioning packages can be applied to a device during initial setup (out-of-box experience or "OOBE") and after ("runtime"). For more information, see [Apply a provisioning package](./provisioning-packages/provisioning-apply-package.md).
```

JohanFreelancer9 commented on windows/configuration/provisioning-packages/provisioning-apply-package.md:
```diff
@@ -20,40 +20,82 @@ manager: dougeby
 
-## During initial setup, from a USB drive
+> [!TIP]
+> In addition to the methods below, you can use the PowerShell cmdlet [Install-ProvisioningPackage](/powershell/module/provisioning/Install-ProvisioningPackage) with `-LogsDirectoryPath` to get logs for the operation.
```
```suggestion
> In addition to the following methods, you can use the PowerShell cmdlet [Install-ProvisioningPackage](/powershell/module/provisioning/Install-ProvisioningPackage) with `-LogsDirectoryPath` to get logs for the operation.
```

JohanFreelancer9 commented on windows/configuration/set-up-shared-or-guest-pc.md:
```diff
@@ -185,30 +185,7 @@ You can configure Windows to be in shared PC mode in a couple different ways:
-On a desktop computer, navigate to **Settings** &gt; **Accounts** &gt; **Work access** &gt; **Add or remove a management package** &gt; **Add a package**, and selects the package to install. 
-
-![add a package option.](images/package.png)
+Provisioning packages can be applied to a device during initial setup (out-of-box experience or "OOBE") and after ("runtime"). For more information, see "[Apply a provisioning package](./provisioning-packages/provisioning-apply-package.md)."
```
```suggestion
Provisioning packages can be applied to a device during initial setup (out-of-box experience or "OOBE") and after ("runtime"). For more information, see [Apply a provisioning package](./provisioning-packages/provisioning-apply-package.md).
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @aisgbnok, kindly commit the minor suggested changes. Many thanks.

aisgbnok commented:
@JohanFreelancer9 Thank you for those suggestions. I viewed a few other articles and your changes do align with the overall documentation style better than what I had. I have committed them accordingly.

aczechowski commented:
Hi @aisgbnok, wow, thanks for such a detailed contribution! 🎉 In this public repo, the best way to get a view of the outcome is on the **Files** tab, use the rich diff option at the top of the markdown file:

![image](https://user-images.githubusercontent.com/26784733/162801833-6e4f1471-608f-4d8b-8269-45aae72e9243.png)

(We do have a preview option available in the private version of this repo, but that's only internally available.)

The person that I'd like to ask to do a deeper review on this is out this week. So I'd like to #hold-off on approving/merging this change until they get back next week. They should also be able to help answer your other questions.

aisgbnok commented:
@aczechowski Thanks, the rendered differences do help a bit! That sounds great, no rush. I look forward to their review! 👍

aisgbnok commented:
@aczechowski Quick question about the docs in general that you might be able to answer if you can spare some time. Currently, I am using traditional markdown syntax for the images, should I instead use:
`:::image type="content" source="memdocs/media/docs-github-edit.png" alt-text="This is the alt text.":::` as [shown here.](https://github.com/MicrosoftDocs/memdocs/blob/main/template.md#images)

And when/how would you use a lightbox? Is that the preferred method for images in Microsoft Docs? Thanks!

aczechowski commented:
@aisgbnok great question, thanks for asking! See the entry in the public contributor guide: https://docs.microsoft.com/contribute/markdown-reference#images, which goes into more detail. Both formats are supported, although I'm generally using the new style myself. I think the main advantage is that it automatically adds a grey border.

In general, always include alt text, and make sure it ends with a period. This practice helps all customers better use the graphics.

A lightbox is an option that's especially useful for larger images with lots of detail. It allows the user to select the image to expand it. So I don't always add a lightbox. Here's an example of it in use: https://docs.microsoft.com/mem/configmgr/comanage/workloads#diagram-for-app-workloads

Another tip: I assume you're using VSCode with the major workflow as noted https://docs.microsoft.com/contribute/get-started-setup-tools. I also strongly recommend use of the Docs Authoring Pack extension. You can then press Alt -M in VSCode to open the Docs Authoring Pack menu, which has a bunch of very useful options. Including "image" that walks you through the process to add images (and uses the docs.ms style `:::`). Hope this helps! Don't hesitate to ask if you have further questions, I'm always happy to help public contributors be more successful.

aisgbnok commented:
@aczechowski Sorry for the slow response. Wow, thank you for the detailed explanation. I didn't know there was a more detailed contributing and style guide. I was looking for one #10468, but kept finding the [Writing Style Guide](https://docs.microsoft.com/style-guide/welcome/) and didn't look further, my bad. 😓

Furthermore, I was not using the Docs Authoring Pack. It's really helpful and coupled with VSCode it can somewhat render the docs locally (including the new image syntax), at least better than IntelliJ which is what I was using before. This is great for seeing how things will be laid out and rendered similar to production. 

I went ahead and replaced the default markdown with the custom docs markdown for the images. Thanks again, I appreciate it!

aczechowski commented:
@aisgbnok some answers on their way to your initial questions. I can respond to the third question regarding ms-settings links - let's try it! I've seen other instances where the feedback-hub protocol handler is used, which I think work ok. So those ms-settings protocol handler links should work as well. That's a really creative use! Once we merge this PR, we can take a look at the published version. Worst case is it doesn't do anything, and then we can followup and remove it.

aisgbnok commented:
@aczechowski Fantastic, no rush! I'm interested to see if the ms-settings links work as well. I based the new paths on Windows 11, so I thought adding an ms-settings link, which will go to the right place regardless of version, might be beneficial. 

I looked at the source for the [ms-settings URI scheme documentation](https://github.com/MicrosoftDocs/windows-uwp/blob/b5e604f6b3e3f5dad3915acd80ae957fd379d8c1/windows-apps-src/launch-resume/launch-settings-app.md?plain=1#L366), but unfortunately, they [aren't actually "live" on the page but just plaintext.](https://docs.microsoft.com/windows/uwp/launch-resume/launch-settings-app#system) When I have some time I'll have to see if I can find an example in one of the existing docs repos. Thanks again!

aisgbnok commented:
I see, it looks like the Feedback Hub pages have a "traditional" URL that triggers the browser to use the feedback-hub protocol handler. 

https://github.com/MicrosoftDocs/windows-insider/blob/130402c0ade78d816e9b20539a2304b97f9a039f/wip/feedback-hub/feedback-hub-app.md?plain=1#L18

https://aka.ms/WIPFeedbackHub

aczechowski commented:
Here's an example of a live page that uses the feedback-hub protocol handler: https://docs.microsoft.com/windows/deployment/upgrade/submit-errors#submit-feedback

`feedback-hub://?referrer=resolveUpgradeErrorsPage&tabid=2&contextid=81&newFeedback=true&feedbackType=2&topic=submit-errors.md`
which at least for me with Edge on Win11, pops a message in the browser about opening the app, and then does so when I select OK. So I assume the same behavior will work with the ms-settings protocol handler, but we'll see.

aczechowski commented:
Adding comments from @gkomatsu to answer your other questions:

For 1st question "Set up a multi-app kiosk on Windows 10 devices"
- Using PPKGs to setup  Windows devices as Kiosk is a common scenario. And AssignedAccess CSP is the CSP that enables the configuration. Adding a link to this doc will help to connect the dots. https://docs.microsoft.com/en-us/windows/client-management/mdm/assignedaccess-csp

For 2nd question, "Clean PC"?
- I am not clear of the question and where CleanPC CSP came from. The note on "computer that has already been set up" refers to a PC that has gone through OOBE and has some configurations applied.

aisgbnok commented:
@aczechowski @gkomatsu Let me explain a little better. 

## Current PR
I am rewriting and updating the "[Apply a provisioning package](https://docs.microsoft.com/windows/configuration/provisioning-packages/provisioning-apply-package)" article. A few other articles have subsections that duplicate this article.

### Kiosk
For example, the ["Set up a multi-app kiosk on Windows 10 devices"](https://docs.microsoft.com/windows/configuration/lock-down-windows-10-to-specific-apps#apply-ppkg) article contains an "Apply provisioning package to device" section. Here are what my changes on that page look like so far:
![image](https://user-images.githubusercontent.com/10838153/167796192-cbd70cb2-6e4e-4958-8b0a-2625480f2c05.png)
I removed the duplicate information and added a link to the main "[Apply a provisioning package](https://docs.microsoft.com/windows/configuration/provisioning-packages/provisioning-apply-package)" article but I kept the note. Should I move the note somewhere into the main "Apply a provisioning package" article, or keep it there as it is specific to that page?

In response to your answer, do you now think I should link to the CSP (https://docs.microsoft.com/en-us/windows/client-management/mdm/assignedaccess-csp) in that note? There is a link to it in the following subsection, so maybe only if the note gets moved?

### Shared PCs
The same applies to the note under the "[Set up a shared or guest PC with Windows 10/11](https://docs.microsoft.com/windows/configuration/set-up-shared-or-guest-pc#apply-the-provisioning-package)" article. 
![image](https://user-images.githubusercontent.com/10838153/167799803-fd25fccb-c382-4027-aa5a-4e4ae9a31245.png)
Does the data loss note apply to all PCs when a provisioning package is being applied or only devices in shared PC mode? You mention "has some configurations applied" does this mean any configuration or specifically something like the SharedPC CSP? Should the note be moved to the main page or kept on that page?

## Alternative

Instead of keeping the notes on their separate pages, maybe include a subsection near the top of the "[Apply provisioning package](https://docs.microsoft.com/windows/configuration/provisioning-packages/provisioning-apply-package)" article titled something like "Considerations" that includes all the notes. Here is a quick draft:
![image](https://user-images.githubusercontent.com/10838153/167805277-22daecd7-19f9-43ec-8757-1e10e4725d77.png)

Sorry for the long explanation, I just wanted to be clear. I'm kind of brainstorming these last few changes, but there is a possibility what I have now is perfectly fine. Feel free to take what I've done into your own hands if that's easier. I like to be confident in my changes when I make them, and I'm not sure what is best moving forward. Thanks again!

aisgbnok commented:
Note: Back merge (c833d7bf8df726112ce875d0ee776bcb0dbf3fa9 and ff6f3e34c26bda84407c8f095ab5e4a6828c4f5e) resolves merge conflict from cab71e43f47442a67b3086124d7c68709d7b0764.

aisgbnok commented:
@aczechowski Hey, apologies for all the confusion. Why don't we have this reviewed and merged as-is? If further improvements should be made I can handle them in a separate PR? I don't want this to get too convoluted and keeping the Notes where they are I think is fine. I was just nit-picking. Let me know what you think! Thanks.

aczechowski commented:
Thanks @aisgbnok for the update, and apologies for the delayed response. I'll go ahead and merge this. If you have further recommendations on how to clarify those notes, I'll look forward to another PR. 😉 And in general, I'm a fan of inter-linking articles to make it easier for folks to find information. (p.s. I might actually wait a little bit to merge this, there's a sync issue right now, and I want to make sure that's clear before I add this to the mix...)

aisgbnok commented:
@aczechowski Thanks for all the help on this! 🎉

aisgbnok commented:
For reference, the `ms-settings` URI links do work as wanted. At least on my end.😀

aczechowski commented:
Same! That's a great tip, I'll add to our internal style guide for future reference/use 😄
